sqlglot/build.py

```diff
@@ -0,0 +1,8 @@
+from sqlglot import expressions as exp
+
+
+def select(*codes, dialect=None, parser_opts=None):
```

Comment:
can we add a from_ here as well, it's common to start with a from statement

Reply:
Yep

Reply:
also not a fan of codes, maybe just expressions is fine

Reply:
Ah yeah good catch, forgot to change this.

FWIW, `code` was copied from `sqlglot.parse`: https://github.com/tobymao/sqlglot/blob/main/sqlglot/__init__.py#L12

Reply:
who wrote that!!! lol, can you refactor that as well

Reply:
that one can probably be sql cuz you can only pass sql in there